pkg/geo/geoindex/testdata/s2_geometry

```diff
@@ -275,8 +275,7 @@ F0/L0/, spilled
 
 d-within distance=2 name=point1
 ----
-[F0/L30/000000000000000000000000000000, F0/L30/033333333333333333333333333333], 
-F0/L0/, F0/L1/2, F0/L2/22, [F0/L30/222000000000000000000000000000, F0/L30/222333333333333333333333333333], 
+[F0/L30/000000000000000000000000000000, F0/L30/333333333333333333333333333333], 
```

Comment:
this output has changed, but since we have "spilled" i think it's functionally equivalent.